### PR TITLE
Return link content instead of returning whole entry in link data api(health-information/informationId)

### PR DESCRIPTION
**Pull request by chethandeshpande**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/wwwroot/openapi.yaml [hunk 1]
```diff
@@ -284,6 +284,62 @@ paths:
...
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: >
+            **Causes:**
+              * Downstream system(s) is down.
+              * Unhandled exceptions.
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/wwwroot/openapi.yaml]: Descriptions are incorrect.

File: src/In.ProjectEKA.HipService/wwwroot/openapi.yaml [hunk 2]
```diff
@@ -284,6 +284,62 @@ paths:
...
+      tags:
+        - data flow
+      summary: Health information link data request
+
+      description: >
+        Return data respective to the link
+      parameters:
+        - $ref: '#/components/parameters/InformationId'
+        - $ref: '#/components/parameters/Token'
+        - $ref: '#/components/parameters/TransactionId'
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/wwwroot/openapi.yaml]: Why do we need transaction Id if we have information id already?

Comment (chethandeshpande) [reply src/In.ProjectEKA.HipService/wwwroot/openapi.yaml]: If we want to identify for which data flow transaction that the link data is being fetched, we can have transactionId

File: src/In.ProjectEKA.HipService/wwwroot/openapi.yaml (hunk 1 above)
Comment (chethandeshpande) [reply src/In.ProjectEKA.HipService/wwwroot/openapi.yaml]: I will fix it :)

File: src/In.ProjectEKA.HipService/wwwroot/openapi.yaml (hunk 2 above)
Comment (ganesanarun) [reply src/In.ProjectEKA.HipService/wwwroot/openapi.yaml]: information id is unique right, we could back track from that always right.

Comment (chethandeshpande) [reply src/In.ProjectEKA.HipService/wwwroot/openapi.yaml]: You are right. Logically, we are not doing anything with the transactionId in this api. we are just returning it back in the response. So, we can remove the transactionId in this api I guess.

**Pull request merged**